index.html

```diff
@@ -166,11 +166,21 @@ <h2 class="about_primary_text">About me</h2>
             </div>
             <div class="about_bottom">
                 <div class="text_box">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
-                <form action="" class="about_form">
-                    <input type="text" placeholder="Full name">
-                    <input type="text" placeholder="Email address">
-                    <textarea name="" id="" cols="30" rows="10">Hey I’d like to connect to...</textarea>
-                    <button class="get_in_touch"><a href="#">Get in touch</a></button>
+                <form action="https://formspree.io/f/{form_id}" class="about_form" method="post">
+                    <ul>
+                        <li>
+                            <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
+                        </li>
+                        <li>
+                            <input type="email" name="email" id="email" placeholder="Email address" required>
+                        </li>
+                        <li>
+                            <textarea name="textarea" id="textarea" maxlength="500" required>Hey I’d like to connect to...</textarea>
+                        </li>
+                        <li>
+                            <button class="get_in_touch"><a href="#">Get in touch</a></button>
+                        </li>
```

Comment:
Great job on implementing this part 💪, One small issue to mention is this button it is meant to submit the form but with the a tag inside it redirects the user to the top of the page instead , I suggest that you remove the a tag  and add type="submit" attribute to the button 
Also please don't forget to add hover and press effects to this button as requested 
![button effects](https://user-images.githubusercontent.com/59707859/228271578-7b1dc5c4-7e9d-4bde-adcb-1ada550ec2ca.PNG)